### ISSUE TITLE
More training  details about tokenizer

### ISSUE THREAD

**Issue by GooThinker** [LanDiff/LanDiff#3]
Very good job. I have a few questions here to ask: 1. What is the cosine similarity between the original features and the features reconstructed by tokenizer? What about mse loss? Can you show the log? Thank you very much! ! 

2. When training tokenizer, will you perform a random latent drop?https://github.com/LanDiff/LanDiff/blob/main/landiff/tokenizer/modules/blocks.py#L937

3. For the tokenizer in this paper, are there any more training tricks to share? Thank you.

**Comment by yinaoxiong** [comment]
Thank you for your attention. 
The MSE loss on the validation set is approximately 0.3. During tokenizer training, we did not randomly drop latent tokens. No additional tricks beyond those presented in the paper were used for tokenizer training. We plan to open-source our tokenizer training code soon; please star our repository and stay tuned.

**Comment by GooThinker** [comment]
When MSE loss=0.3, what is the cos similarity between features? Are high-dimensional features very difficult to reconstruct? Looking forward to reply, thank you~

**Comment by GooThinker** [comment]
In addition, how many steps did the tokenizer model finally train?

**Comment by yinaoxiong** [comment]
> In addition, how many steps did the tokenizer model finally train?

We trained the tokenizer model for a total of 130k steps.
